.github/workflows/app-logging.yml

```diff
@@ -2,15 +2,17 @@ name: PIMS Logging CI
 
 on:
```

Comment:
Seems unrelated to the PR

Reply:
What's the purpose of the Action?

Reply:
Just a few changes I made and pass to doug

Reply:
The purpose is to run and test the logging part of the app front and backend. 
https://github.com/bcgov/PSP/tree/dev/openshift/4.0/templates/Logging

This is not now I just made a chanage